example_bytecode_test.go

```diff
@@ -1,4 +1,4 @@
-package duktape_test
+package duktape
```

Comment:
Can we change the package back to duktape_test? Keeping example code in a different package forces you to write like a user.